torchvision/datasets/utils.py

```diff
@@ -95,7 +95,10 @@ def download_url(url, root, filename=None, md5=None):
                 )
             else:
                 raise e
-
+        # check integrity of downloaded file
+        if not check_integrity(fpath, md5):
+            print("Error downloading file: Dataset not found in " + fpath + " or corrupted.")
```

Comment:
1. This is not only used to download datasets. Maybe change to wording from "Dataset" to "file"?
2. Is there a possibility that the file is not present after the download? If not, I would remove the "not found" part

Reply:
1. Yes, totally agree. 
2. `check_integrity` checks first whether a file is present or not. So I included that information in  the log message. However, I'm not sure if [urlretrieve](https://docs.python.org/3/library/urllib.request.html#urllib.request.urlretrieve) would always create a file. It also seems to be a deprecated function.